### PR TITLE
Notes and corrections to the readme

### LINKED ISSUE #194: Typo in README.md
> In practical terms, it pulls in the official package-set as a base, and you are then able to add any package that might not be in the package set, or override esisting ones.

There is a typo here. It should be "existing" instead of "esisting".

### LINKED ISSUE #196: Error: Remote host not found on `spago build` inside container
Versions:
- `+ spago@0.7.7`
- `+ purescript@0.12.5`

Problem:

I am attempting to run `spago build` installed via npm inside of a container. When I do that on an empty project after running `spago init` I get the following error. The same command run on the host (mac osx) works fine. Also running `curl https://raw.githubusercontent.com/purescript/package-sets/psc-0.12.5-20190427/src/mkPackage.dhall` in the host container works just fine. That seems to indicate that the problem lies in how spago is resolving the host. 

```
$> npx spago build
spago: 
↳ ./packages.dhall
  ↳ https://raw.githubusercontent.com/purescript/package-sets/psc-0.12.5-20190427/src/mkPackage.dhall sha256:0b197efa1d397ace6eb46b243ff2d73a3da5638d8d0ac8473e8e4a8fc528cf57

Error: Remote host not found


      https://raw.githubusercontent.com/purescript/package-sets/psc-0.12.5-20190427/src/mkPackage.dhall sha256:0b197efa1d397ace6eb46b243ff2d73a3da5638d8d0ac8473e8e4a8fc528cf57


/iogine/packages.dhall:112:7
```

### PR DESCRIPTION
**Pull request by ericbmerritt**
### Description of the change

Adds a note about `netbase` to the installation section, fixes a few spelling mistakes and adds some comma's for clarity.

### PR checklist:

- [ ] Updated the version number - didn't due because it's a minor change to the readme
- [ ] Added the change to the "Unreleased" section of the changelog.- see above
- [ ] Added some example of the new feature to the `README` - see above
- [ ] Added a test for the contribution (if applicable) - see above

**P.S.**: the above checks are not compulsory to get a change merged, so you may skip them. However, taking care of them will result in less work for the maintainers and will be much appreciated 😊

Fix #194 
Fix #196

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -71,6 +71,8 @@ The recommended installation methods for Windows, Linux or macOS are:
 **Note:** we assume you already installed the [PureScript compiler][purescript].
 If not, get it with `npm install -g purescript`, or the recommended method for your OS.
 
+**Note:** If you get networking errors in ubuntu you may need to install the `netbase` package with `apt install netbase`.
+
```
Comment (f-f) [inline README.md]: We already have [a FAQ question](https://github.com/spacchetti/spago#im-getting-weird-errors-about-libtinfoso5) about other Ubuntu installation problems, so we should either move this to a FAQ question or move that suggestion up here (either is fine by me)

**Comment by ericbmerritt** [comment]
Absolutely, my emacs was being overly helpful.

File: README.md (hunk 1 above)
Comment (ericbmerritt) [reply README.md]: Good call. I will fix it.

File: README.md [hunk 2]
```diff
@@ -681,3 +681,7 @@ So make sure `build` works first. If you still get a failure you might be encoun
 [spago-issues]: https://github.com/spacchetti/spago/issues
 [dhall-hash-safety]: https://github.com/dhall-lang/dhall-lang/wiki/Safety-guarantees#code-injection
 [spago-latest-release]: https://github.com/spacchetti/spago/releases/latest
+
+#### I getting a 'Host Not Found' error running `spago install` or `spago build` in ubuntu
+
+If you get networking errors in ubuntu you may need to install the `netbase` package with `apt install netbase`.
```
Comment (f-f) [inline README.md]:
```suggestion
If you get networking errors in ubuntu you may need to install the `netbase` package with `apt install netbase`. For more details see [this issue](https://github.com/spacchetti/spago/issues/196).
```

Comment (ericbmerritt) [reply README.md]: fixed

**Pull request merged**